apps/remix-ide/src/app/components/side-panel.tsx

```diff
@@ -19,18 +18,17 @@ const sidePanel = {
 
 // TODO merge with vertical-icons.js
```

Comment:
export class VerticalIcons extends Plugin {

Reply:
what do you mean?

Reply:
ah not true, should be a plugin I removed that old comment

Reply:
:)) I 'pasted' wrong line. comment was about todo